cpan/Encode/Unicode/Unicode.pm

```diff
@@ -259,7 +259,7 @@ Consider that "division by zero" of Encode :)
 =head1 SEE ALSO
 
 L<Encode>, L<Encode::Unicode::UTF7>, L<http://www.unicode.org/glossary/>,
-L<http://www.unicode.org/unicode/faq/utf_bom.html>,
+L<http://www.unicode.org/faq/utf_bom.html>,
```

Comment:
This file is maintained on CPAN: https://github.com/dankogai/p5-encode

Reply:
https://github.com/dankogai/p5-encode/pull/158